src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -225,21 +231,25 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
                             {!!(!this.store.cancerTypes.isPending && !this.store.cancerStudies.isPending) && (
                                 <Observer>
                                     {() => {
+                                    const studyLimitReached = this.store.selectableSelectedStudyIds.length > 50;
+                                    const multipleReferenceGenomes = (!this.checkForSingleReferenceGenome);
+                                    let tooltipMessage = <span>Open summary of selected studies in a new window.</span>;
+                                    if (studyLimitReached) {
+                                        tooltipMessage = <span>Studies selected for summary exceeded limit of 50</span>;
+                                    }
+                                    if (multipleReferenceGenomes) {
+                                        tooltipMessage = <span>Studies selected came from different reference genomes</span>;
+                                    }
 
-                                        const studyLimitReached = (this.store.selectableSelectedStudyIds.length > 50);
-                                        const tooltipMessage = studyLimitReached ?
-                                            <span>Too many studies selected for study summary (limit: 50)</span> :
-                                            <span>Open summary of selected studies in a new window.</span>;
-
-                                        return (
+                                    return (
                                             <DefaultTooltip
                                                 placement="top"
                                                 overlay={tooltipMessage}
                                                 disabled={!this.store.summaryEnabled}
                                                 mouseEnterDelay={0}
                                             >
 
-                                                <Button bsSize="xs" disabled={studyLimitReached} bsStyle="primary"
+                                                <Button bsSize="xs" disabled={studyLimitReached||multipleReferenceGenomes} bsStyle="primary"
```

Comment:
lets take this condition out and assign it to a local variable of the method just to make it super clear.